src/commands/Unban.ts

```diff
@@ -64,17 +67,33 @@ async function unbanUserFromRooms(
         continue;
       }
       if (rule.test(member.userID)) {
+        let logMessage = `Unbanning ${member.userID} in ${revision.room.toRoomIDOrAlias()}`;
+        if (invite) {
+          logMessage += ", and re-inviting them.";
+        }
         await managementRoomOutput.logMessage(
           LogLevel.DEBUG,
           "Unban",
-          `Unbanning ${member.userID} in ${revision.room.toRoomIDOrAlias()}`,
+          logMessage,
           revision.room.toRoomIDOrAlias()
         );
```

Comment:
Do you think that this would be simpler? 

```typescript
await managementRoomOutput.logMessage(
  LogLevel.DEBUG,
  "Unban",
  `Unbanning ${member.userID} in ${revision.room.toRoomIDOrAlias()}` + invite ? ', and re-inviting them." : '',
  revision.room.toRoomIDOrAlias()
);
```

Reply:
Yeah it would be. I was just trying to stick to a line length limit and figured the string in place was quite long lol. I'll replace it.